SU-SWS/stanford_profile
# D8CORE-6242 D8CORE-4977 D8CORE-6055 Improve people lists and add pronoun field

# READY FOR REVIEW

# Summary
- Adjust people content titles in the lists to be headers and in the proper order of headers
- Added a pronoun field to people and display on the node page.

# Need Review By (Date)
- :shrug: 

# Urgency
- low

# Steps to Test
1. checkout this branch
2. import configs
3. create some people content
4. view the /people page
5. verify the people titles are `h2` tags
6. view one of the people term pages like "Staff"
7. verify the group headings are `h2` tags while the people titles are `h3` tags.
